.shippable.yml

```diff
@@ -0,0 +1,18 @@
+build_image: freedomjs/freedom
+language: node_js
+node_js:
+- '0.10'
+notifications:
+  email:
+    recipients:
+      - freedom@cs.washington.edu
+    on_success: change
+    on_failure: always
+before_install:
+  - "export DISPLAY=:10.0"
+  - "Xvfb :10 -screen 0 1280x1024x24 &"
+install:
+  - "git pull"
```

Comment:
I prefer to have a custom image so that we can control the version number (stable/dev/canary)